RockWeb/Blocks/Finance/BusinessList.ascx

```diff
@@ -30,8 +30,8 @@
                             <Rock:RockTemplateField>
                                 <HeaderTemplate>Contact Information</HeaderTemplate>
                                 <ItemTemplate>
-                                    <%# Eval("PhoneNumber") %><br />
-                                    <%# Eval("Email") %>
+                                    <%# Eval("PhoneNumber") %>  <%# !string.IsNullOrEmpty( (string)Eval( "PhoneNumber" ) )? "<br />":string.Empty %>
+                                    <%# Eval("Email") %> <%# string.IsNullOrEmpty( (string)Eval( "PhoneNumber" ) )? "<br />&nbsp; ":string.Empty %>
```

Comment:
@azturner I think this was part of the requirements to always have that column have two lines so all the rows would be the same height (most will have both a phone and email).